akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -72,6 +73,11 @@
         Self addHeaders(Iterable<HttpHeader> headers);
 
         /**
+         * Returns a copy of this message with the given http credential header added to the list of headers.
+         */
+        Self addCredentials(HttpCredentials credentials);
```

Comment:
Have fixed it and verified with `mimaReportBinaryIssues`, correct me if there exists something wrong in my code, thanks!